.github/workflows/airflow-periodic-tests.yml

```diff
@@ -0,0 +1,77 @@
+name: Airflow Periodic Integration Tests
+
+on:
+  schedule:
+    - cron: "0 21 * * 4" # Run at 9PM PST on every Thursday
+  workflow_dispatch:
+
+jobs:
+  run-airflow-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 480
+    name: SDK Integration Tests against Airflow Compute
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: ./.github/actions/setup-server
+        timeout-minutes: 7
+
+      # TODO(ENG-2537): Use our separate GH actions credentials.
+      - uses: ./.github/actions/fetch-test-config
+        with:
+          aws_access_key_id: ${{ secrets.KENNY_AWS_ACCESS_KEY_ID }}
+          aws_secret_access_key: ${{ secrets.KENNY_AWS_SECRET_ACCESS_KEY }}
+          s3_test_config_path: periodic-airflow-test-config.yml
+      
+      - name: Start Airflow cluster
+        working-directory: scripts/compute
+        run: ./airflow_test_setup.sh &
+      
+      - name: Wait for Airflow cluster to be ready
+        run: while ! echo exit | nc localhost 8000; do sleep 1; done
+      
+      - name: Get Airflow password
+        run: |
+          airflow_password=$(cat ~/airflow/standalone_admin_password.txt)
+          echo "airflow_password=$airflow_password" >> $GITHUB_ENV
+      
+      - name: Update Airflow password in test-credentials.yml
+        working-directory: integration_tests/sdk
+        # NOTE (Saurav): This replaces all password fields under the compute section.
+        # This is safe to do because only the test_airflow compute is being used
+        # in this test. Otherwise the logic gets more complicated.
+        run: sed -i '/^compute:/,/^[^ ]/ s/^\(\s*password:\s*\).*/\1${{ env.airflow_password }}/' test-credentials.yml
+
+      - name: Install any data connector packages
+        run: |
+          aqueduct install s3
+          aqueduct install snowflake
+
+      - name: Run the SDK Integration Tests
+        working-directory: integration_tests/sdk
+        run: pytest aqueduct_tests/ -rP -vv -n 1
+
+      - uses: ./.github/actions/upload-artifacts
+        if: always()
+        with:
+          prefix: Airflow Compute
+
+      # Sets it as an environmental variable.
```

Comment:
I'll uncomment before merging

Reply:
Actually will leave this commented out until I finish triaging all of the test case failures. Otherwise there will be a lot of noise generated. See https://linear.app/aqueducthq/issue/ENG-3009/investigate-airflow-integration-test-failures-and-triage-accordingly